packages/agent/src/canisterStatus/index.ts

```diff
@@ -0,0 +1,245 @@
+/** @module CanisterStatus */
```

Comment:
Not sure if `canisterStatus` is a good name. `time` and `subnet` are not tied to the canister. Maybe call it `StateTree` to match with the interface spec?

Reply:
My reasoning is that `StateTree` sounds overly generic - state tree of what? The entire IC? 

The feature was motivated by me wanting to discover the ModuleHash, Metadata, Controllers, and Subnet of a canister. I'd like the naming to reflect that motivation, which should make it more discoverable.

the fact that this involves a state tree in a certificate is more or less an implementation detail, from my POV as a developer